data-loading/Makefile

```
@@ -70,6 +70,16 @@ data/start-solr-backup: data/setup-complete
 data/check-solr-backup:
 	curl 'http://localhost:8983/solr/name_lookup/replication?command=details'
 
+# Step 6. Shutdown the Solr instance.
+### data/stop-solr:
+###    	docker exec name_lookup solr stop -p 8983 -verbose
+
+# Step 7. Generate the backup tarball.
+data/backup:
+	mkdir -p data/var/solr/data
+	mv /var/solr/name_lookup_shard1_replica_n1/data/snapshot.backup data/var/solr/data
+	cd data && tar zcvf snapshot.backup.tar.gz var
```

```suggestion
	cd data && tar zcvf snapshot.backup.tar.gz var && touch $@
```

If you're using a phony target, you should touch it at the end of the process so that it's clear that the process completed successfully. You can do that by adding `touch $@` at the end of the command.